skimage/registration/_lddmm.py

```diff
@@ -0,0 +1,2211 @@
+"""
+This is an implementation of the LDDMM algorithm with modifications, written
+by Devin Crowley and based on "Diffeomorphic registration with intensity
+transformation and missing data: Application to 3D digital pathology of
+Alzheimer's disease."
+
+This paper extends on an older LDDMM paper, "Computing large deformation
+metric mappings via geodesic flows of diffeomorphisms."
+
+This is the more recent paper:
+Tward, Daniel, et al. "Diffeomorphic registration with intensity
+transformation and missing data: Application to 3D digital pathology of
+Alzheimer's disease." Frontiers in neuroscience 14 (2020).
+https://doi.org/10.3389/fnins.2020.00052
+
+This is the original LDDMM paper:
+Beg, M. Faisal, et al. "Computing large deformation metric mappings via
+geodesic flows of diffeomorphisms." International journal of computer vision
+61.2 (2005): 139-157.
+https://doi.org/10.1023/B:VISI.0000043755.93987.aa
+"""
+
+import warnings
+import numpy as np
+from collections import namedtuple
+
+from scipy.interpolate import interpn
+from scipy.linalg import inv, solve, det, svd
+from skimage.transform import resize, rescale
+
+from ._lddmm_utilities import _validate_scalar_to_multi
+from ._lddmm_utilities import _validate_ndarray
+from ._lddmm_utilities import _validate_spacing
+from ._lddmm_utilities import _compute_axes
+from ._lddmm_utilities import _compute_coords
+from ._lddmm_utilities import _multiply_coords_by_affine
+from ._lddmm_utilities import _compute_tail_determinant
+from ._lddmm_utilities import resample
```

Comment:
```suggestion
```

Reply:
the function is not used if I'm not mistaken

Reply:
Wow, I think I actually fully deprecated resample without noticing. skimage.transform.resize is quicker and slightly cruder but it's not strictly necessary to be exact in the rescaling for multiscales so I just went with it.